app/assets/javascripts/hooch.js

```diff
@@ -595,16 +598,64 @@ var initHooch = function(){
     }),
     HistoryPusher: Class.extend({
       init: function($history_pusher){
-        this.key = $history_pusher.data('key')
-        this.value = $history_pusher.data('value')
+        this.$history_pusher = $history_pusher
+        this.getPusherType()
+        var history_pusher = this
+        this.bindPusher()
+      },
+      bindPusher: function(){
+        var bind_method = 'bind' + this.pusher_type
+        this[bind_method]()
+      },
+      bindLink: function(){
         var history_pusher = this
-        $history_pusher.on('click', function(){
+        this.$history_pusher.on('click apiclick', function(){
```

Comment:
I thought it was a custom event type I created so that hooch could interact with thin_man and know the difference between a user click and an even-triggered click. But now I can't find it anywhere, I must have deprecated it. Anyhow, it won't hurt anything for now and we can refactor all the references in tech debt sometime.